20_data_munge.yml

```diff
@@ -0,0 +1,23 @@
+target_default: 20_data_munge
```

Comment:
I don't think this part of the pipeline has been fully updated to do flow instead of temp.

Reply:
probably fine for now though

Reply:
This step is updated for flow. It only collapses UV data down to daily data, though. There is no script to do what you want, though, which is to extract number of obs per year through time. However, the `20_data_munge/out/daily_temperatures.rds.ind` target should give you what you need to make that calculation.

Reply:
I guess I don't really understand why `temperature` is used here at all? There is no temperature data in this pipeline yet as far as I am aware and based on what you've added in this PR.

Reply:
haha DOH! I even wrote that out and didn't catch it. Yes, it's modified, but I forgot to change temperature to flow.